__tests__/components/AnonymousChat/child/ChatItemMyTextV2.test.tsx

```diff
@@ -1,43 +1,178 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable global-require */
+/* eslint-disable react/display-name */
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

These lines disable ESLint rules for the entire file. It's generally a bad practice to disable linting rules across an entire file, as it can lead to code inconsistencies and potential bugs. If you need to disable a rule, it's better to do it on a line-by-line basis.

```diff
- /* eslint-disable @typescript-eslint/no-var-requires */
- /* eslint-disable global-require */
- /* eslint-disable react/display-name */
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->